service/static/js/index.js

```diff
@@ -7,36 +7,67 @@ $(function () {
   function update_wishlist_form_data(res) {
     $("#wishlist-id").val(res.id);
     $("#wishlist-name").val(res.name);
-    $("#wishlist-user_id").val(res.user_id);
+    $("#wishlist-user-id").val(res.user_id);
   }
 
   function update_product_form_data(res) {
-    $("#wishlist-product_id").val(res.id);
-    $("#wishlist-inventory_product_id").val(res.inventory_product_id);
-    $("#wishlist-product_name").val(res.name);
-    $("#wishlist-product_price").val(res.price);
-    $("#wishlist-product_status").val(res.status);
-    $("#wishlist-product_pic").val(res.pic_url);
-    $("#wishlist-product_description").val(res.short_desc);
+    $("#wishlist-product-id").val(res.id);
+    $("#wishlist-inventory-product-id").val(res.inventory_product_id);
+    $("#wishlist-product-name").val(res.name);
+    $("#wishlist-product-price").val(res.price);
+    $("#wishlist-product-status").val(res.status);
+    $("#wishlist-product-pic").val(res.pic_url);
+    $("#wishlist-product-description").val(res.short_desc);
+  }
+
+  // Creates the results table
+  function add_row(record, results_table) {
+    const id = record.id || '';
+    const name = record.name || '';
+    const user_id = record.user_id || ''
+    const product_list = JSON.stringify(record.products) || ''
+    const row = "<tr><td class=\"results-table__wishlist-id col-md-2\">"+id+"</td><td class=\"col-md-4\">"+name+
+      "</td><td class=\"col-md-2\">"+user_id+"</td><td class=\"col-md-6\">"+product_list +"</td></tr>";
+    results_table.append(row);
+  }
+
+  function build_table(data) {
```

Comment:
It wasn't a valid table before: you `append`ed to the `div`, not the `table`, so the`<table>` element and a bunch of `<tr>`s were sibilings under `<div class="table-responsive" id="search-results">`.